doc/en/download/Assertions.rst

```diff
@@ -98,3 +98,16 @@ Required files:
 test_plan.py
 ++++++++++++
 .. literalinclude:: ../../../examples/Assertions/Plotly/test_plan.py
+
+
+Marking
```

Comment:
We might need a bit of documentation that explain the use case we try to solve, that may point to this good example.

Reply:
Sure thing.

Reply:
Added below. Please opine.